### DISCUSSION TITLE
Brand guidelines for Babel logo usage

### DISCUSSION THREAD

**Discussion by benhalverson** [babel/babel#11731] [General] [answered]
Hello,
Where can I find the branding guidelines for Babel?
The nodejs.dev website dev team would like to use Babel's logo on what will eventually be a new homepage.
See this issue. https://github.com/nodejs/nodejs.dev/issues/637

**Comment by nicolo-ribaudo** [comment, marked as answer]
Hi 👋

We don't currently have any guidelines, but:
1. Please use the logo from https://github.com/babel/logo, which is the "source of truth"
2. If you need an icon version, you can use https://github.com/babel/website/issues/2028 (we'll add it to the `babel/logo` repository)
